fe/fe-core/src/test/java/com/starrocks/planner/MaterializedViewTest.java

```diff
@@ -2655,7 +2655,7 @@ public void testRangePredicates2() {
             testRewriteOK(mv, "select deptno as col1, empid as col2, emps.locationid as col3 from emps " +
                     " left join locations on emps.locationid = locations.locationid where emps.deptno < 10 or emps.deptno > 10");
             // TODO: support OrPredicates as Range Predicates.
-            testRewriteFail(mv, "select deptno as col1, empid as col2, emps.locationid as col3 from emps " +
+            testRewriteOK(mv, "select deptno as col1, empid as col2, emps.locationid as col3 from emps " +
                     " left join locations on emps.locationid = locations.locationid where emps.deptno < 5 or emps.deptno > 20");
         }
```

Comment:
Greate job! can you add more complex tests here to display we can support `or`  predicates?

Reply:
add some case in MaterializedViewTest#testRangePredicates3.
you can reply here if want to add some case.